README.md

```diff
@@ -73,6 +73,8 @@ Hash as specified by [stylelint](https://github.com/stylelint/stylelint/blob/mas
 
 doesn't accept `files` option
 
+when `syntax` option is `""` or `null`, stylesheets will be parsed with all of the supported syntaxes: `css|sass|scss|less|sugarss`
```

Comment:
i think we can just say lints all supported stylelint languages by default and if you want to scope it down to one language you can specify syntax